src/map/MapRoutePath.js

```diff
@@ -1,28 +1,13 @@
 import { useTheme } from '@mui/styles';
 import { useId, useEffect } from 'react';
-import { useSelector } from 'react-redux';
 import { map } from './core/MapView';
```

Comment:
Can you please check the warnings and errors.

Reply:
I don't have any warnings